artview/components/nav.py

```diff
@@ -0,0 +1,271 @@
+        groupBox = QtGui.QGroupBox("File Navigation")
+        gBox_layout = QtGui.QGridLayout()
+
+        self.firstbutton = QtGui.QPushButton("First")
+        self.firstbutton.setToolTip("Load first file in directory")
+        self.firstbutton.clicked.connect(self.goto_first_file)
+        gBox_layout.addWidget(self.firstbutton, 0, 0, 1, 1)
+
+        self.prevbutton = QtGui.QPushButton("Previous")
+        self.prevbutton.setToolTip("Load previous file")
+        self.prevbutton.setIconSize(QtCore.QSize(32,32))
+        self.prevbutton.clicked.connect(self.goto_prev_file)
+        gBox_layout.addWidget(self.prevbutton, 0, 1, 1, 1)
+
+        self.nextbutton = QtGui.QPushButton("Next")
+        self.nextbutton.setToolTip("Load next file")
+        self.nextbutton.clicked.connect(self.goto_next_file)
+        gBox_layout.addWidget(self.nextbutton, 0, 2, 1, 1)
+
+        self.lastbutton = QtGui.QPushButton("Last")
+        self.lastbutton.setToolTip("Load last file in directory")
+        self.lastbutton.clicked.connect(self.goto_last_file)
+        gBox_layout.addWidget(self.lastbutton, 0, 3, 1, 1)
+
+        groupBox.setLayout(gBox_layout)
+
+        return groupBox
+
+    def createNavToolbar(self):
+        '''Mount the file navigation toolbar.'''
+        parentdir = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+        groupBox = QtGui.QGroupBox("File Navigation")
+        gBox_layout = QtGui.QGridLayout()
+
+        self.navtoolbar = QtGui.QToolBar()
+        pixfirst = QtGui.QPixmap(os.sep.join([parentdir, 'icons',"arrow_go_first_icon.png"]))
+        pixprev = QtGui.QPixmap(os.sep.join([parentdir, 'icons',"arrow_go_previous_icon.png"]))
+        pixnext = QtGui.QPixmap(os.sep.join([parentdir, 'icons',"arrow_go_next_icon.png"]))
+        pixlast = QtGui.QPixmap(os.sep.join([parentdir, 'icons',"arrow_go_last_icon.png"]))
+        self.act_first = self.navtoolbar.addAction(
+            QtGui.QIcon(pixfirst),
+            "First file: %s"%(self.Vfilelist.value[0]),
+            self.goto_first_file)
+        self.act_prev = self.navtoolbar.addAction(
+            QtGui.QIcon(pixprev),
+            "Previous file: %s"%(self.Vfilelist.value[self.fileindex - 1]),
+            self.goto_prev_file)
+        self.act_next = self.navtoolbar.addAction(
+            QtGui.QIcon(pixnext),
+            "Next file: %s"%(self.Vfilelist.value[self.fileindex + 1]),
+            self.goto_next_file)
+        self.act_last = self.navtoolbar.addAction(
+            QtGui.QIcon(pixlast),
+            "Last file: %s"%(self.Vfilelist.value[-1]),
+            self.goto_last_file)
+
+        gBox_layout.addWidget(self.navtoolbar)
+        groupBox.setLayout(gBox_layout)
+
+        return groupBox
+
+    def createInfoUI(self):
+        '''Mount the information text.'''
+        groupBox = QtGui.QGroupBox("File Information")
+        gBox_layout = QtGui.QGridLayout()
+
+        self.dirIn = os.path.dirname(self.Vradar.value.filename)
```

Comment:
there is no guaranty that the "filename" attr exist, and I don't think the working directory need to be the directory of the current open radar. It happens to be, but it could be different in some special configuration.